docs/source/analysis-workflows/upgrade-cost-analysis.rst

```diff
@@ -116,6 +119,50 @@ Submit the pipeline with JADE
 
 If everything succeeds, it produces same aggregated upgrade tables in ``output-stage1``.
 
+Generic Workflow
+----------------
+Let's assume that you have multiple networks defined in OpenDSS model files where each network has
+its own ``Master.dss``.
+
+- ``./custom_models/model1/Master.dss``
+- ``./custom_models/model2/Master.dss``
+
+Single Execution Mode
+~~~~~~~~~~~~~~~~~~~~~
+1. Configure the simulation parameters and in an input JSON file called ``upgrades.json``.
+Refer to this
+`file <https://github.com/NREL/disco/blob/main/tests/data/test_upgrade_cost_analysis_generic_vreg.json>`_
```

Comment:
can we change this to file <https://github.com/NREL/disco/blob/main/tests/data/test_upgrade_cost_analysis_generic.json>

The other one is an extra one I added to check for multiple vregs. We could potentially remove it, so better to just point to the "generic.json"